src/components/AuthorList.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+export default class AuthorList extends React.Component {
+  render() {
+    // Render all authors
+    let renderAuthors = () => {
```

Comment:
Instead of making this a function, just do:
```
let authorNodes = _.map(authors || [])......
``` 

and then at the bottom render

```
{authorNodes}
```